packages/svelte-check/src/writers.ts

```diff
@@ -159,10 +163,25 @@ export class MachineFriendlyWriter implements Writer {
                     : null;
 
             if (type) {
-                const { line, character } = range.start;
+                const { start, end } = range;
                 const fn = JSON.stringify(filename);
                 const msg = JSON.stringify(message);
-                this.log(`${type} ${fn} ${line + 1}:${character + 1} ${msg}`);
+                if (this.isVerbose) {
+                    this.log(
+                        JSON.stringify({
+                            type,
+                            filename,
+                            start,
+                            end,
+                            message,
+                            code,
+                            codeDescription,
+                            source
+                        })
+                    );
+                } else {
+                    this.log(`${type} ${fn} ${start.line + 1}:${start.character + 1} ${msg}`);
+                }
             }
```

Comment:
```suggestion
                const { start, end } = range;
                if (this.isVerbose) {
                    this.log(
                        JSON.stringify({
                            type,
                            filename,
                            start,
                            end,
                            message,
                            code,
                            codeDescription,
                            source
                        })
                    );
                } else {
                    const fn = JSON.stringify(filename);
                    const msg = JSON.stringify(message);
                    this.log(`${type} ${fn} ${start.line + 1}:${start.character + 1} ${msg}`);
                }
            }
```

Reply:
Holy hell, that's a lot taller than I thought it'd be.

TL;DR: Move the `JSON.stringify`s inside the else branch

Reply:
Good suggestion...did it